### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by dylancorvidae**

### CODE REVIEW

File: src/components/containers/Main.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  }
+
+  render() {
+    return (
+      <>
+        <form>
+          <UrlInputForm handleUrlChange={this.handleUrlChange} />
+          <div id="methods">
+            
+            <label for="get"><Method id={'get'} handleMethodChange={this.handleMethodChange} checked="checked" />GET</label>
```
Comment (ridenouram) [inline src/components/containers/Main.js]: labels can't have a "for" property, so this is creating a console error

> webpack-internal:///./node_modules/react-dom/cjs/react-dom.development.js:530 Warning: Invalid DOM property `for`. Did you mean `htmlFor`?
    in label (created by Main)
    in div (created by Main)
    in form (created by Main)
    in Main (created by App)
    in section (created by App)
    in main (created by App)
    in App

File: src/components/form/Method.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styles from '../../styles/method.css';
+
+export default class Method extends Component {
```
Comment (ridenouram) [inline src/components/form/Method.js]: each radio button should not be its own component. Rather, "method" should be stored in state and changing the radio button selection updates state

File: src/services/apiCall.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+
+// function apiCall(url, method, json) {
+function apiCall(url, method, json) {
+  if(method === 'get' || method === 'post' || method === 'put' || method === 'delete') {
+    return fetch(url, { method }) 
+      .then(res => res.json())
+      .catch(err => {
+        console.log(err);
+      });
+  } else {
```
Comment (CapyJara) [inline src/services/apiCall.js:10]: not sure that you need this whole part. i think you could just pass the method and json into the fetch and you should be fine.

File: src/components/containers/Main.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    this.setState({ urlInput: target.value });
+  }
+
+  handleMethodChange = ({ target }) => {
+    this.setState({ method: target.value });
+  }
+
+  handleInputJsonChange = ({ target }) => {
+    this.setState({ inputJson: target.value });
+  }
```
Comment (CapyJara) [inline src/components/containers/Main.js:25]: handleMethodChange = ({ target }) => {
    this.setState({ [target.name]: target.value });
  }

yours are totally fine! but if you dont want to write a bunch of them, you can use this for all of them as long as your input tags have name attributes on them that match up with your state object.

**Pull request merged**